editor/plugin.js

```diff
@@ -1,16 +1,16 @@
  /**
- *
- * Plugin to create mcq question
- * @extends org.ekstep.contenteditor.questionUnitPlugin
- * @author Jagadish P <jagadish.pujari@tarento.com>
- */
- org.ekstep.contenteditor.questionUnitPlugin.extend({
- 	
- 	initialize: function() {  
- 		ecEditor.addEventListener("org.ekstep.plugins.mcqplugin:showpopup", this.loadHtml, this);
- 		var templatePath = ecEditor.resolvePluginResource(this.manifest.id, this.manifest.ver, 'editor/templates/horizontal_template.html');
- 		var controllerPath = ecEditor.resolvePluginResource(this.manifest.id, this.manifest.ver, 'editor/controllers/horizontal_controller.js');
- 		ecEditor.getService(ServiceConstants.POPUP_SERVICE).loadNgModules(templatePath, controllerPath);
- 	}
+  *
+  * Plugin to create mcq question
+  * @extends org.ekstep.contenteditor.questionUnitPlugin
+  * @author Jagadish P <jagadish.pujari@tarento.com>
+  */
+ org.ekstep.questionunitMTF = {};
+ org.ekstep.questionunitMTF.EditorPlugin = org.ekstep.contenteditor.questionUnitPlugin.extend({
+   initialize: function() {
+     ecEditor.addEventListener("org.ekstep.plugins.mcqplugin:showpopup", this.loadHtml, this);
```

Comment:
Is this event being used at all?

Reply:
No not needed we are calling from question unit . please check